addon/components/chart-component.js

```diff
@@ -28,19 +28,27 @@ export default Ember.Component.extend(ColorableMixin, ResizeHandlerMixin, {
   horizontalMarginRight: null,
 
   /**
+   * An array of the values in the data that is passed into the chart
+   * @type {Array<Number>}
+   */
+  allValues: Ember.computed('finishedData.@each.value', function() {
```

Comment:
I feel like this name is a little too generic. Maybe `allFinishedDataValues` or something?